src/common/services/api/donations.service.js

```diff
@@ -44,10 +44,27 @@ function DonationsService( cortexApiService ) {
       .pluck( 'gifts' );
   }
 
+  function getReceipts( data ) {
+    return cortexApiService
+      .post( {
+        path: '/receipts/items',
+        followLocation: true,
+        data: data
+      } )
+      .map( (response) => {
+        angular.forEach(response['receipt-summaries'], (item, index) => {
+          item['pdf-link'] = response['links'][index+1];
```

Comment:
Relying on this index seems like it could be a little brittle... If another link gets added it'll be off by one. I guess the elements should be in the same order but idk if that is guaranteed.

It might be better to create that link with the `transaction-number`. Idk. What do you think? I know you asked about it at one point.